helpers/query_setter_helper.go

```diff
@@ -0,0 +1,34 @@
+package helpers
```

Comment:
The file name is kind of long. To make it more generic and concise, what do you think of `helpers/database_helper.go` since it could be used for other database queries helpers 💭